modules/client/src/test/java/org/apache/ignite/client/fakes/FakeSchemaRegistry.java

```diff
@@ -67,53 +69,68 @@ public static void setLastVer(int lastVer) {
 
     /** {@inheritDoc} */
     @Override
-    public SchemaDescriptor schema(int ver) {
-        if (ver == 0) {
+    public SchemaDescriptor schemaNow(int version) {
```

Comment:
The postfix was added because we have a legacy: `schema()` was actually blocking, so if we still call it `schema()`, but make it work instantly, without any waits, this might confuse colleagues that are used to the old meaning. `schemaNow()` is chosen by analogy with `CompletableFuture`, where there is a blocking `get()` and 'instant' `getNow(...)`; but this name causes confusion.

Would `schemaImmediately()` be better? It would solve that 'two meanings of 'now'' problem, at least.

Reply:
We personally discussed renaming.